gossip/components/transaction_requester.c

```diff
@@ -199,7 +199,8 @@ retcode_t get_transaction_to_request(requester_state_t *const state,
     memset(hash, FLEX_TRIT_NULL_VALUE, FLEX_TRIT_SIZE_243);
   }
 
-  if (((double)rand() / (double)RAND_MAX) < state->conf->p_remove_request) {
+  if (((double)rand() / (double)RAND_MAX) < state->conf->p_remove_request &&
```

Comment:
could you format this more nicely? also could you add a TODO/FIXME wrt. abstracting the rand() call here?